port/win/port_win.h

```diff
@@ -241,6 +241,8 @@ extern void InitOnce(OnceType* once, void (*initializer)());
 #define CACHE_LINE_SIZE 64U
 #endif
 
+#define CACHE_ALIGNED(size) __attribute__((__aligned__(size)))
```

Comment:
folly is using `__declspec(align(size))` for msvc. Shall we follow?
https://github.com/facebook/folly/blob/e7e6d29a00b685ab673fd76250ca3354088de46b/folly/Portability.h#L39-L42